src/backend/TrafficCourts/Citizen.Service/Models/Dispute/NoticeOfDispute.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Text.Json.Serialization;
+using TrafficCourts.Common.Converters;
+
+namespace TrafficCourts.Citizen.Service.Models.Dispute
+{
+    public class NoticeOfDispute
```

Comment:
The models I have and gave to Shark have comments for all these properties and validation attributes.

Reply:
Added comments and validation attributes to the properties of NoticeOfDispute model in Citizen API.